codeforboston/cliff-effects
# Add .nvmrc file. Fixes #931

Test plan:

- [Install NVM](https://github.com/creationix/nvm#installation)
- Run `nvm install` from the project root directory
- It should install and use the latest LTS version of Node (at the moment, this is `v10.13.0`)
- Open a new terminal and go to the project directory
- Run `nvm use`
- It should switch to use the LTS version (verify that it is with `node --version`)
